lib/matplotlib/artist.py

```diff
@@ -249,6 +249,23 @@ def get_window_extent(self, renderer):
         """
         return Bbox([[0, 0], [0, 0]])
 
+    def get_clip_bbox_extents(self):
```

Comment:
Perhaps call this `get_clip_extents`? (otherwise it's not clear whether it's just the extents of the clip box or of the intersection of the clip_box and the clip_path)

Reply:
How about `get_clip_bbox`.  I guess this should also be private, just so we don't have to worry about changing it.

Reply:
+1 for private as usual ;)
get_clip_bbox sounds too close to get_clip_box to me...

Reply:
OK, private and `_get_clipping_extent_bbox` was what I decided on.  It is an extent (rather than the actual clip path) and it returns a `Bbox`.